src/Resources/contao/dca/tl_templates.php

```diff
@@ -227,7 +227,7 @@ public function addNewTemplate()
 		foreach ($kernel->getContainer()->get('contao.resource_provider')->getResourcesPaths() as $strModule => $strFolder)
 		{
 			// Continue if there is no templates folder
-			if ($strModule == 'repository' || !is_dir($strFolder))
```

Comment:
I think this is wrong. Isn't `$strFolder` the Contao resource path? Should it not be `$strFolder . '/templates`?